### ISSUE TITLE
Missing non-expired self parent in `FCM-MultiSBReconnect-3R-1k-20m`

### ISSUE THREAD

**Issue by alittley** [hashgraph/hedera-services#10116]
Exception occurs in `Shadowgraph`:

```
Missing non-expired self parent for (0,6223,11c62181) sp(0,6142,7fe1967a) op(5,6222,8b13025b)
```

- Failure is consistent. [data folder](http://35.247.76.217:8095/swirlds-automation/develop/10N/MultiSBReconnect/20231127-055648-GCP-Daily-MultiSBReconnect-3R-10N/FCM-MultiSBReconnect-3R-1k-20m/)

Presumably related to https://github.com/hashgraph/hedera-services/pull/9747

**Comment by alittley** [comment]
The following series of events is what is causing this behavior:

1. Event `X` goes through a node's intake pipeline, and is added to the Shadowgraph
2. The node falls behind
3. In preparation for reconnect, the node clears the Shadowgraph. Notably, the new intake pipeline components do not have their internal state cleared.
4. Once reconnect is complete, event `X` is still non-ancient for the node. It is not part of the shadowgraph, which was cleared, so event `X` is requested via sync from a peer
5. The event deduplicator receives event `X`, but discards it as a duplicate event, since `X` traversed the intake pipeline prior to the reconnect
6. When a child of `X` is added to the Shadowgraph, an exception is logged, since `X` isn't in the Shadowgraph

To solve this issue, I propose simply not clearing the Shadowgraph prior to a reconnect. The intake queue will still be cleared, so no events will be moving through the system. But the Shadowgraph will maintain internal state, and won't have to re-request events that had previously been received and processed.

**Comment by alittley** [comment]
New proposed solution:

- Make internal state of all new intake components clearable
- Create a `PlatformCoordinator`, which is responsible for clearing and flushing wiring components
    - It is anticipated that this coordinator will assume additional responsibilities in the future

**Comment by alittley** [comment]
Update:

After making the proposed changes, the error in the issue body still arises. The root cause of this is that the old event intake would add ancient-non-expired events to the shadowgraph, whereas the new event intake discards all ancient events. In fact, this message should not be an *error*, since it is permissible for a node to be missing ancient events, even if they aren't expired.

Therefore, I've demoted the message to a warning.
